cve_bin_tool/extractor.py

```diff
@@ -84,7 +84,7 @@ def extract_file_rpm(cls, filename, extraction_path):
             else:
                 with popen_ctx(["rpm2cpio", filename], stdout=subprocess.PIPE) as proc:
                     return subprocess.call(
-                        ["cpio", "-idmv"], stdin=proc.stdout, cwd=extraction_path
+                        ["cpio", "-idm", "--quiet"], stdin=proc.stdout, cwd=extraction_path
```

Comment:
This sounds reasoanble.  If it doesn't cause any of our current test cases to fail, we should take it.  Can you go run black on your code so it'll actually pass the first continuous integration test, please?